html/templates/combatant.html

```diff
@@ -4,7 +4,7 @@
 <span class="ac" title="Armor Class" data-bind="text: Combatant.AC"></span>
 <span class="tags">
 <!--ko foreach: Combatant.Tags() -->
-    <span class="tag">
+    <span class="tag" data-bind="visible: Visible">
```

Comment:
It may be better to use the `if` binding here as `data-bind="if: Visible"`.

The `if` binding will completely skip rendering the element, whereas `visible` just puts `display:none` on it as a style.

Reply:
OK, I'll give that a try